zerver/models/groups.py

```diff
@@ -60,6 +60,7 @@ class NamedUserGroup(UserGroup):  # type: ignore[django-manager-missing] # djang
     )
 
     realm_for_sharding = models.ForeignKey("zerver.Realm", on_delete=CASCADE, db_column="realm_id")
+    deactivated = models.BooleanField(default=False)
```

Comment:
Can we use `db_default=False`? I don't see much downside, and it would simplify running the migration to add the field.

Reply:
Updated.